src/sntools/genevts.py

```diff
@@ -77,10 +102,18 @@ def parse_command_line_options():
 
     parser.add_argument("input_file", help="Name or common prefix of the input file(s). Required.")
 
+    # comment in place of a possible new argument to specify pre SN fluxes needed
+    choices = ("sn", "presn")
+
+    parser.add_argument("--mode", metavar="MODE", choices=choices, default=choices[0], 
+                        help="Mode of operation: supernova burst or pre supernova. Choices: %(choices)s. Default: %(default)s.")
```

Comment:
```suggestion
    choices = ("ccsn", "presn")
    parser.add_argument("--mode", metavar="MODE", choices=choices, default=choices[0], 
                        help="Mode of operation: supernova burst or pre supernova. Choices: %(choices)s. Default: %(default)s.")
```
I think it’d be better to be consistent with the terminology used in snewpy.

(This may require a few changes elsewhere in the PR, where this string is hardcoded.)